cockatrice/src/dlg_settings.cpp

```diff
@@ -392,6 +399,7 @@ void AppearanceSettingsPage::retranslateUi()
     tableGroupBox->setTitle(tr("Table grid layout"));
     invertVerticalCoordinateCheckBox.setText(tr("Invert vertical coordinate"));
     minPlayersForMultiColumnLayoutLabel.setText(tr("Minimum player count for multi-column layout:"));
+    maxFontSizeForCardsLabel.setText(tr("Maximum size font for displaying card attributes"));
```

Comment:
Isn't it `font size` ?!

I would also consider adding `on the table`, or maybe better: rephrase to `information displayed on cards`.